test/e2e/aks.go

```diff
@@ -223,8 +223,8 @@ func WaitForAKSSystemNodePoolMachinesToExist(ctx context.Context, input WaitForC
 }
 
 // GetAKSKubernetesVersion gets the kubernetes version for AKS clusters.
-func GetAKSKubernetesVersion(ctx context.Context, e2eConfig *clusterctl.E2EConfig) (string, error) {
-	e2eAKSVersion := e2eConfig.GetVariable(AKSKubernetesVersion)
+func GetAKSKubernetesVersion(ctx context.Context, e2eConfig *clusterctl.E2EConfig, varName string) (string, error) {
```

Comment:
Would it be better to rename `varName` to something more descriptive like `version`, unless the variable needs to be more generic?

Reply:
Yeah this could definitely be more clear. Since this does represent literally the name of a variable, would something like `versionVar` be a bit better? I'll update the doc comment here too.

Reply:
Yup I think that would be perfect!